src/model/wallet.js

```diff
@@ -831,3 +805,112 @@ export const loadSeedPhraseAndMigrateIfNeeded = async id => {
     return null;
   }
 };
+
+async function extractSecretsForWallet(wallet) {
```

Comment:
I can no longer brain, so just making a note: extractSecretsForWallet will currently technically also hold on to the old seed phrase if it's there yes?

Reply:
yes because the old key has the same key name (except the new one is prefixed)